.buildkite/pipeline.yml

```diff
@@ -374,3 +374,5 @@ steps:
   - label: "Publish to serverless"
     branches: main
     trigger: "agentless-serverless-release"
+    build:
+      commit: "${BUILDKITE_COMMIT}"
```

Comment:
wonder if you need to pass the branch too?

i.e., https://github.com/elastic/beats/blob/9ed47e242424679333106dd823a175ba81b5dd6e/.buildkite/pipeline.yml#L121-L127